### DISCUSSION TITLE
2024-04-11 Remix 改变了我编写 dialog 的方式

### DISCUSSION THREAD

**Discussion by giscus** [ThaddeusJiang/thaddeusjiang.github.io#69] [Announcements]
# 2024-04-11 Remix 改变了我编写 dialog 的方式

关于 dialog（对话框），回想一下，我的代码经历过好几次变化，使用 React 之前，React 初级阶段，目前部分项目使用原生 HTML 部分项目使用 Remix framework。

https://thaddeusjiang.com/2024-04-11-remix-gai-bian-liao-wo-bian-xie-dialog-de-fang-shi

**Comment by addlistener** [comment]
这确实是一个很有意思的问题。和我的经验很相似
